moveit_planners/pilz_industrial_motion_planner/package.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>pilz_industrial_motion_planner</name>
+  <version>0.4.10</version>
+  <description>MoveIt plugin to generate industrial trajectories PTP, LIN, CIRC and sequences thereof.</description>
+
+  <maintainer email="a.gutenkunst@pilz.de">Alexander Gutenkunst</maintainer>
+  <maintainer email="c.henkel@pilz.de">Christian Henkel</maintainer>
+  <maintainer email="i.martini@pilz.de">Immanuel Martini</maintainer>
+  <maintainer email="j.schleicher@pilz.de">Joachim Schleicher</maintainer>
+  <maintainer email="h.slusarek@pilz.de">Hagen Slusarek</maintainer>
+
+  <license>BSD</license>
+
+  <url type="website">https://wiki.ros.org/pilz_trajectory_generation</url>
+  <url type="bugtracker">https://github.com/PilzDE/pilz_industrial_motion/issues</url>
```

Comment:
Should these websites be changed to MoveIt? Do you continue to host the source mainly in your organization or here at ros-planning? In the former case, why to merge into MoveIt at all. In this case, it would suffice to provide binary-releases as well as some tutorials/documentation on the MoveIt side.